Add detailed help

djhi opened the pull request:
Add support for specific command help:

- `eydia help`
- `eydia help todo`
- `eydia help todo add`

- [x] Support for commands help
- [x] Add help for the `init` command
- [x] Find a way to avoid `/todo/add/help` route as it forbids adding an `help` todo (solution may be to have a very cryptic internal route for help such as `___help___`)

Commit:
Add detailed help

Commit:
Fix tests

Commit:
Better internal route to avoid reserved help keyword

Commit:
Add init command help

Kmaschta commented on packages/cli/src/default-flags.js:
```diff
@@ -0,0 +1,17 @@
+const DefaultFlags = () => (
+    <>
+        <CommandHelp.Flag name="cwd">
+            Sets the directory into which to execute the command. Default is the
```
This sentence seems weird to me, but I can't find any better one, so..

djhi replied:
Agreed. Couldn't find any better either...

djhi replied:
ping @fzaninotto ?

Kmaschta commented on packages/cli/src/default-flags.js:
```diff
@@ -0,0 +1,17 @@
+            current one.
+        </CommandHelp.Flag>
+        <CommandHelp.Flag name="project">
+            Sets the project name. Default is the current gir repository if
```
s/gir/git/

sedy-bot committed:
Typo fix s/gir/git/

As requested by Kmaschta at https://github.com/marmelab/eydia/pull/8#discussion_r275260614

djhi committed:
Wording
